src/Common/tests/System/IO/Compression/LocalMemoryStream.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
I agree that some of those changes might not be necessary, but they already exist in master and it will make it easier, in the future, to port more changes with minimal effort (It is harder to go through every single line on those files than porting the projects where applicable and removing 2.* specific benchmarks, so I just moved the files).